src/utils/reducers/marker.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Create a marker relevant to a queue item (track).
+ */
+export const markerCreate = function(state, action) {
```

Comment:
I'm starting the process of breaking up the reducers in the redux file up. Those functions are getting out of control!